docs/from_the_sources.md

```diff
@@ -103,13 +105,35 @@ That's it! Go on your browser to the xo-server IP address, and it works! :)
 
 ## Misc
 
-- You can also consider using [forever](https://github.com/nodejitsu/forever) to have always the process running.
+- You can use [forever](https://github.com/nodejitsu/forever) to have the process always running:
 
 ```
 $ yarn global add forever
 $ forever start bin/xo-server
 ```
 
+- Or you can use  [forever-service](https://github.com/zapty/forever-service) to install XO as a service, so it starts automatically at boot. As root:
+
+```
+yarn global add forever
+yarn global add forever-service
+cd /home/username/xo-server/bin/
+forever-service install orchestra -r username -s xo-server
+```
+
+Be sure to change the usernames to the user you are running xo-server as. The forever-service command must be run in the xo-server bin directory. Now you can manage the service, and it will start on boot with the machine:
+
+```
+service orchestra start
+service orchestra status
+```
+
+If you need to delete the service:
+
+```
+forever-service delete orchestra
+```
+
```

Comment:
I wonder if it's not too specific. I mean it's a way like another to do it. Should we use this one @julien-f ?

Reply:
It's in the *Misc* section and correctly documented, I think we can keep it if @Fohdeesha accepts to answer user questions related to it 😉

Reply:
haha agreed!